src/cljs/webchange/editor_v2/translator/translator_form/audio_assets/audios_list/audios_list_item/views.cljs

```diff
@@ -131,17 +138,22 @@
                             (not @edit-state?))
                    [audio-menu {:on-edit         handle-edit
                                 :on-bring-to-top handle-bring-to-top
+                                :on-clear-selection handle-clear-selection
                                 :on-delete       handle-delete}])])))
 
 (defn audios-list-item
-  [{:keys [url alias start duration selected? target]}]
-  (let [handle-change-data (fn [data] (re-frame/dispatch [::translator-form.scene/update-asset url data]))
+  [audio-data]
+  (let [
+        {:keys [url alias start duration selected? target]} audio-data
```

Comment:
Why not destruct in params block?

```
(defn audios-list-item
  [{:keys [url alias start duration selected? target] :as audio-data}]
```

and probably in this case it would be better to rename [`audio-data`](https://github.com/VisionsGlobalEmpowerment/webchange/pull/265/files#diff-764c611a1b865b9eb08f2ce5f0d4378cR158) var in the end on the `let`
block